### PR TITLE
Survey clone: include source props as destination props draft

### PR DESCRIPTION
**Pull request by SteRiccio**
## Description

When cloning a survey, cloned items props will be stored as "draft" props (survey info, node defs, cateories, taxonomies).

## Types of changes

- [ ] Docs change
- [ ] Refactoring
- [ ] Dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How has this been tested

The change has been tested by checking that the props_draft columns were filled, while the props one remained empty.

#### Do you consider this PR needs further testing?

- [ ] No
- [x] Yes

This could be tested with integration tests by checking the values stored into the database after the import.

### CODE REVIEW

**Comment by minotogna** [comment]
Task linked: [CU-hf7k6b Clone - include source props as destination propsDraft](https://app.clickup.com/t/hf7k6b)

File: server/db/utils/transformCallback.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const transformCallback = (row, draft = false, assocPublishedDraft = fals
 
   const rowUpdated = A.pipe(
     // Assoc published and draft properties based on props
-    (rowCurrent) => (assocPublishedDraft ? _assocPublishedDraft(rowCurrent) : rowCurrent),
+    (rowCurrent) => (backup || assocPublishedDraft ? _assocPublishedDraft(rowCurrent) : rowCurrent),
```
Comment (minotogna) [inline server/db/utils/transformCallback.js:17]: is this included in arena-server ?

Comment (SteRiccio) [reply server/db/utils/transformCallback.js:17]: uhm, no, also in arena-server I should include (maybe) a change to the Job class

Comment (minotogna) [reply server/db/utils/transformCallback.js:17]: why a change to the job class?

Comment (SteRiccio) [reply server/db/utils/transformCallback.js:17]: to set inner job parameters/context into the parent job context

Comment (minotogna) [reply server/db/utils/transformCallback.js:17]: can you open a PR then ?

Comment (SteRiccio) [reply server/db/utils/transformCallback.js:17]: it's not needed anymore for Job, I will open a PR for transformCallback only

**Review by minotogna — APPROVED** [review]

**Pull request merged**